test/resources/case2_uninstall_ns/case2-uninstall-ns.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       serviceAccountName: governance-policy-spec-sync
       containers:
       - name: uninstall-ns
-        image: "quay.io/open-cluster-management/governance-policy-spec-sync:latest"
+        image: quay.io/open-cluster-management/governance-policy-spec-sync:latest
```

Comment:
In the future, it'd be nice to not include things that aren't related to your PR and that don't affect the behavior of the code at all.